entries/entry04.md

```diff
@@ -1,8 +1,28 @@
 # Entry 4
-##### X/X/XX
+##### 2/9/20
+## Recap
+  This past month In class, I have been searching for more templates, mostly animations and character templates so I can edit, add on and implement my own animations. Outside of class, I have been coding and tinkering in the Unity engine and also trying out Unity projects and also searched and watched any videos on how controller support works.
 
-Text
 
-[Previous](entry03.md) | [Next](entry05.md)
 
-[Home](../README.md)
\ No newline at end of file
+## Engineering Design Process
+In the Engineering Design Process, I thought I was close and ready to move onto step 5 which is **prototyping**, but I took a few steps back and stayed in the **planning process** which is step 4. I feel like I want to plan even more than I already did. I did not feel confident creating my prototype just yet. I will be reaching the prototyping stage soon but I just want to make sure that things go smoothly in the beginning and I can get more work done rather than thinking too much .
+
+
+## Knowledge
```

Comment:
What I'm really looking for here are things like screenshots, code snippets, and if possible, interactive examples (I know that's hard with unity, which is why you should really include the first two) as evidence of learning. This is a pretty slim entry for 1 whole month's worth of learning.